lib/wash/exe/mount.stream.js

```diff
@@ -177,23 +179,29 @@ var CreateStreamsResult;
  * @return {!Promise<!CreateStreamsResult>}
  */
 var createStreams_ = function(cx, streamType, streamSrc) {
-  if (streamType === 'websocket') {
-    var streams = new WebSocketStreams();
-    return streams.open(streamSrc).then(
-      function() {
-        /** @type {!CreateStreamsResult} */
-        var result = {
-          readableStream: streams.readableStream,
-          writableStream: streams.writableStream,
-          close: function() { streams.close(); },
-          resume: function() { streams.resume(); }
-        };
-        return Promise.resolve(result);
-      }
-    );
-  }
-
-  // TODO(rpaquay): Add support for more stream types.
-  return Promise.reject(
-      new AxiomError.Invalid('stream-type', streamType));
+  var streams;
+
+  return Promise.resolve().then(function() {
+    switch (streamType) {
+      case 'websocket':
+        streams = new WebSocketStreams();
+        return streams.open(streamSrc).then(function() {return "ws"});
```

Comment:
I didn't understand about the `;`, but here's a couple others: 1) we use single quotes for strings; 2) I'd keep the vocabulary to a minimum by sticking to `'websocket'` instead of `'ws'`.